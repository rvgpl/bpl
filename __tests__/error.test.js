import React from 'react';
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer';
import Error from "../pages/_error";

it('Error text exists', () => {
    const app = shallow(
      <Error />
    )
    expect(app.find('h1').exists());
  })

it('Error renders', () => {
  const component = renderer.create(<Error />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
