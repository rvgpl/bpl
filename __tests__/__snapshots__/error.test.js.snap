exports[`test Error renders 1`] = `
<div>
  <h1
    className="f2 lh-solid fw6 mw6 mt5 center tc">
     
    An error has occured on Client
  </h1>
</div>
`;
